javascript/extractor/src/com/semmle/js/extractor/ExprKinds.java

```diff
@@ -77,6 +77,9 @@
     binOpKinds.put("**", 87);
     binOpKinds.put("**=", 88);
     binOpKinds.put("??", 107);
+    binOpKinds.put("&&=", 116);
+    binOpKinds.put("||=", 117);
+    binOpKinds.put("??=", 118);
```

Comment:
We'll need to update `CFGExtractor` as the control-flow graph for these operators is not like other binary operators; the right-hand side is only conditionally evaluated.